ballerina-test-automation/gradle.properties

```diff
@@ -1,7 +1,7 @@
-swan-lake-latest-version=swan-lake-2201.3.2
+swan-lake-latest-version=swan-lake-2201.3.3
 swan-lake-latest-spec-version=2022R4
-swan-lake-latest-version-display-text=2201.3.2
-swan-lake-latest-tool-version=1.3.11
-latest-tool-version=1.3.11
+swan-lake-latest-version-display-text=2201.3.3
+swan-lake-latest-tool-version=1.3.13
+latest-tool-version=1.3.13
```

Comment:
Do we need to update the tool version?

Reply:
@NipunaMadhushan

Reply:
As the tool version of the 3.x branch is 1.3.11, no need to update here https://github.com/ballerina-platform/ballerina-distribution/blob/309cd02851884a1dae22a178b7e6b9b8fe04065b/gradle.properties#L64
@udda1996 Please confirm

Reply:
we can update the tool version here as well. since it affects the installer tests let's update the `ballerinaCommandVersion` to 1.3.13 in 3.x branch as well

Reply:
@NipunaMadhushan Then we need to update the installer tests in the 3.x branch as well. Could you please send a PR for that?